q2_feature_table/_rename.py

```diff
@@ -0,0 +1,71 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2020, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import warnings
+
+import biom
+import qiime2
+
+
+def _generate_new_names(old_ids, rename, strict, verbose=False):
+    """
+    Checks the list of samples ot be renamed
+    """
+    # all old ids need to be unique
+    if (rename.value_counts() > 1).any():
+        raise ValueError('All new ids must be unique.\n'
+                         'Try qiime feature-table group if you want '
+                         'to combine multipel samples in the same table.')
```

Comment:
```suggestion
                         'to combine multiple samples in the same table.')
```

Also, related to my other comment about samples vs features - can we come up with another word to use here, instead of samples?

Reply:
I think I replaced this with ids